cost/aws/instance_cloudwatch_utilization/aws_instance_cloudwatch_utilization.pt

```diff
@@ -472,20 +483,46 @@ end
 policy "pol_utilization" do
   validate_each $ds_inefficient_utilization_metrics do
     summary_template "{{ rs_project_name }} (Account ID: {{ rs_project_id }}): {{ len data }} AWS instances with inefficient utilization"
-    detail_template <<-EOS
-# AWS Instance Utilization
-| Region | instanceID | instanceType | Platform | Hostname | Private DNS Name | CPU Maximum % | CPU Average % | Memory Maximum % | Memory Average % | Next Instance Size % |
-| ------ | ---------- | ------------ | -------- | -------- |---------------- | ------------- | ------------- | ---------------- | ---------------- | ------------------- |
-{{ range data -}}
-| {{.region}} | {{.instanceId}} | {{.instanceType}} | {{.platform}} | {{.hostname}} |  {{.privateDnsName }} | {{ .cpu_maximum }} | {{ .cpu_average }} | {{if eq .mem_maximum "101"}}N/A{{else}}{{.mem_maximum}}{{end}} | {{if eq .mem_average "101"}}N/A{{else}}{{.mem_average}}{{end}} | {{ .next_instance_size }} |
-{{ end -}}
-___
-###### Policy Applied in Account: {{ rs_project_name }} (Account ID: {{ rs_project_id }}) within Org: {{ rs_org_name }} (Org ID: {{ rs_org_id }})
-EOS
     escalate $email
     check logic_or(eq($param_avg_used_memory, -1), gt(to_n(val(item,"mem_average")), $param_avg_used_memory))
     check logic_or(eq($param_avg_cpu, -1), gt(to_n(val(item,"cpu_average")),$param_avg_cpu))
 	escalate $esc_downsize_instances
+    export do
+      resource_level true
+      field "region" do
+        label "Region"
+      end
+      field "id" do
```

Comment:
This doesn't HAVE to change, but it's less changes in the template to use path here instead of changing the instance_id everywhere in the template.

>       field "id" do
>        label "Instance ID"
>        path jmes_path(item, "instanceId")
>      end

Reply:
Sure, I upcoming tickets I'll use path

Reply:
after seeing errors when using the above jmes_path engineering said it was implmented, but we can use a string instead.

path "volumeId".

I updated the file with the correct syntax.  FYI for future policies